packages/ui-kit/src/components/code-block.tsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
-import { css, ThemeProvider, useTheme } from '@emotion/react';
+import { css, Theme, ThemeProvider, useTheme } from '@emotion/react';
```

Comment:
Types should be imported using the `type` keyword.

```suggestion
import type { Theme } from '@emotion/react';
```

The latest TS versions also support a more advanced syntax:

```suggestion
import { css, type Theme, ThemeProvider, useTheme } from '@emotion/react';
```